### PR TITLE
added docs in AggregateRoots, ValueObjects and entities

### PR DESCRIPTION
**Pull request by tauraigombera**
I have added doc files for the following:
-Aggregate Roots: Constituency, GroupVillageHead, and TraditionalAuthority
-Entities: PollingCenter, Village, and ZipCode
-ValueObjects: Geolocation

### CODE REVIEW

File: Docs/AggregateRoots/Constituency.AggregateRoot.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    public void UpdateMalePopulation(long population)
+    public void UpdateFemalePopulation(long population)
+    public void UpdateGeolocation(double latitude, double longitude)
+    public void AddWard(Guid wardId)
+    public void RemoveWard(Guid wardId)
+    /*----------------------------*/
+    public Constituency GetConstituencyById(Guid id)
+    public List<Constituency> GetBatchConstituenciesById(List<Guid> ids)
+    public Constituency SearchConstituencyByName(string name)
+    public List<Constituency> GetConstituency()
```
Comment (liwoo) [inline Docs/AggregateRoots/Constituency.AggregateRoot.md]: `GetConstituencies` - pluralize

File: Docs/AggregateRoots/GroupVillageHead.AggregateRoot.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    public void UpdateMalePopulation(long population);
+    public void UpdateFemalePopulation(long population);
+    public void UpdateGeolocation(double latitude, double longitude);
+    public void AddVillage(Guid villageId);
+    public void RemoveVillage(Guid villageId);
+    /*----------------------------*/
+    public GroupVillageHead GetGroupVillageHeadById(Guid id);
+    public List<GroupVillageHead> GetBatchGroupVillageHeadsById(List<Guid> ids);
+    public GroupVillageHead SearchGroupVillageHeadByName(string name);
+    public List<GroupVillageHead> GetGroupVillageHead();
```
Comment (liwoo) [inline Docs/AggregateRoots/GroupVillageHead.AggregateRoot.md]: pluralize here as well

File: Docs/AggregateRoots/TraditionalAuthority.AggregateRoot.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    public void UpdateMalePopulation(long population)
+    public void UpdateFemalePopulation(long population)
+    public void UpdateGeolocation(double latitude, double longitude)
+    public void AddGroupVillageHead(Guid groupVillageHeadId)
+    public void RemoveAddGroupVillageHead(Guid groupVillageHeadId)
+    /*----------------------------*/
+    public TraditionalAuthority GetTraditionalAuthorityById(Guid id)
+    public List<TraditionalAuthority> GetBatchTraditionalAuthoritiesById(List<Guid> ids)
+    public TraditionalAuthority SearchTraditionalAuthorityByName(string name)
+    public List<TraditionalAuthority> GetTraditionalAuthority()
```
Comment (liwoo) [inline Docs/AggregateRoots/TraditionalAuthority.AggregateRoot.md]: pluralize

File: Docs/Entities/Village.Entity.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+```
+
+## Example Payload
+
+```json
+{
+  "id": "00000000-0000-0000-0000-000000000000",
+  "name": "string",
+
```
Comment (liwoo) [inline Docs/Entities/Village.Entity.md]: no gap here

File: Docs/ValueObjects/Geolocation.ValueObject.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Geolacation Value Object
+
+## Description
+
+Geolacation is a position of something on the Earth's surface determined by latitude, longitude, and sometimes altitude.
+
+## Methods
```
Comment (liwoo) [inline Docs/ValueObjects/Geolocation.ValueObject.md]: you can remove the `methods` section

**Review by liwoo — CHANGES_REQUESTED** [review]

**Comment by tauraigombera** [comment]
Resolved the requested changes

**Review by liwoo — APPROVED** [review]

**Pull request merged**